recipes/apache-libcloud/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.18.0" %}
+
+package:
+  name: apache-libcloud
+  version: {{ version }}
+
+source:
+  fn: apache-libcloud-{{ version }}.tar.bz2
+  url: http://apache.osuosl.org/libcloud/apache-libcloud-{{ version }}.tar.bz2
+  sha1: fbf558d7e52336042dd9a456dec3a73f4b35e38e
+
+build:
+  number: 0
```

Comment:
You can use the following here instead of `bld.bat` and `build.sh`
```
build:
[...]
  script: python setup.py install --single-version-externally-managed --record record.txt
```